src/init.cpp

```diff
@@ -55,6 +56,10 @@ template <>
 struct Init<Backend::MC> {
   static void initialize(const configuration& cfg) {
     memory::internal::initializeUmpireHostAllocator(cfg.umpire_host_memory_pool_initial_bytes);
+    // install mpi polling loop
+    // pika::mpi::experimental::init(false);
+    pika::mpi::experimental::init(false, true);
```

Comment:
It's not required for pika to initialize MPI, is it? In e.g. CP2K we're not going to get to initialize it anyway, so I would avoid doing it here.